src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue26083.cs

```diff
@@ -0,0 +1,33 @@
+﻿#if WINDOWS || MACCATALYST //Window resizing behavior is only present on Mac Catalyst and Windows platforms.
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue26083 : _IssuesUITest
+	{
+		public Issue26083(TestDevice testDevice) : base(testDevice)
+		{
+		}
+
+		public override string Issue => "UI not updating GridItemsLayout when Span becomes 1";
+
+		[Test]
+		[Category(UITestCategories.CollectionView)]
+		public void ItemsWrapGridShouldUpdateBasedOnCollectionViewSize()
```

Comment:
Pending snapshot on Mac and Windows:
**ItemsWrapGridShouldUpdateBasedOnCollectionViewSize**

<img width="1010" height="729" alt="image" src="https://github.com/user-attachments/assets/e01cb45d-64df-4566-a44f-708fac1d12f2" />

Could you commit the images?